core/templates/pages/review-test-page/review-test-page-root.component.html

```diff
@@ -0,0 +1,16 @@
+<oppia-angular-root>
+  <oppia-base-content>
+    <navbar-breadcrumb>
+      <div class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak">
+        <span class="oppia-navbar-breadcrumb-separator"></span>
+        <span translate="I18N_REVIEW_TEST_PAGE_BREADCRUMB"></span>
+      </div>
+    </navbar-breadcrumb>
+    <content>
+      <review-test-page></review-test-page>
+    </content>
+    <page-footer></page-footer>
+  </oppia-base-content>
+</oppia-angular-root>
+
+<script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=default"></script>
```

Comment:
Do we need this? (IIRC, this is only needed for the editor's page but review-test page is to be used by learners.)

Reply:
It was used in the .mainpage.html so I included it at the end. I was not sure weather it is needed or not